chapter_optimization/momentum_vn.md

```diff
@@ -96,10 +97,10 @@ This allows us to realize most of the benefits of averaging over a batch without
 We will revisit this averaging procedure in more detail later.
 -->
 
-Giá trị $\beta$ lớn tương ứng với trung bình trong khoảng rộng, trong khi đó giá trị $\beta$ nhỏ có nghĩa là chỉ có một chút chỉnh sửa nhẹ so với phương pháp gradient bình thường.
-Gradient mới này không còn trỏ về hướng đi dốc nhất trong từng trường hợp cụ thể nữa mà thay vào đó đi theo hướng trung bình có trọng số của các gradient trước đó.
-Điều này cho phép chúng ta nhận được hầu hết lợi ích của việc tính toán trung bình theo batch mà không phải tốn chi phí tính toán gradients theo cả batch.
-Chúng ta sẽ xem xét lại quy trình lấy trung bình một cách cụ thể hơn ở những phần sau.
+Khi $\beta$ lớn ta lấy trung bình trong khoảng rộng, trong khi đó nếu $\beta$ nhỏ phương pháp này sẽ không khác nhiều so với hạ gradient thông thường.
+Gradient mới này không còn có hướng đi dốc nhất trong từng trường hợp cụ thể nữa mà thay vào đó đi theo hướng trung bình có trọng số của các gradient trước đó.
+Điều này giúp chúng ta nhận thêm lợi ích của việc tính trung bình theo batch mà không cần tốn chi phí tính toán gradients trên batch. <!-- câu này hơi khó hiểu, mn góp ý với -->
```

Comment:
Ý của câu này là với `trung bình rò rỉ` thì cũng giống như thực hiện GD (phương pháp lấy trung bình gradient của toàn bộ tập dữ liệu) mà vẫn giữ được những đặc tính của minibatch SGD

Reply:
```suggestion
Điều này giúp chúng ta nhận thêm lợi ích của việc tính trung bình theo batch mà không cần tốn chi phí tính toán gradients trên batch.
```